File: fix_export_config.py
import subprocess

result = subprocess.run(['git', 'show', 'HEAD:src/constants/seriesExportConfig.ts'], capture_output=True)
content = result.stdout.decode('utf-8')

content = content.replace(
    "import { BRANDING } from '@/config/branding';",
    "import { BRANDING } from '@/config/branding';\nimport type { AudienceProfile } from '@/constants/audienceConfig';\nimport { resolveAudienceProfile } from '@/constants/audienceConfig';"
)

content += "\n\nexport function resolveExportTerminology(\n  audienceProfile?: AudienceProfile\n): { assemblyLabel: string; participantLabel: string } {\n  const profile = resolveAudienceProfile(audienceProfile);\n  return {\n    assemblyLabel:    profile.assembly,\n    participantLabel: profile.participant,\n  };\n}\n"

with open('src/constants/seriesExportConfig.ts', 'w', encoding='utf-8', newline='\n') as f:
    f.write(content)

print('Done')

File: fix_export_config.js
const fs = require('fs');
const { execSync } = require('child_process');

const raw = execSync('git show HEAD:src/constants/seriesExportConfig.ts').toString();
const content = raw.replace(/\r\n/g, '\n');

const withImports = content.replace(
  "import { BRANDING } from '@/config/branding';",
  "import { BRANDING } from '@/config/branding';\nimport type { AudienceProfile } from '@/constants/audienceConfig';\nimport { resolveAudienceProfile } from '@/constants/audienceConfig';"
);

const withFunction = withImports + `
export function resolveExportTerminology(
  audienceProfile?: AudienceProfile
) {
  const profile = resolveAudienceProfile(audienceProfile);
  return {
    assemblyLabel:    profile.assembly,
    participantLabel: profile.participant,
  };
}
`;

fs.writeFileSync('src/constants/seriesExportConfig.ts', withFunction, { encoding: 'utf8' });
console.log('Done - lines: ' + withFunction.split('\n').length);
